test/GovernanceStorage.t.sol

```diff
@@ -764,15 +764,16 @@ contract GovernanceStorageTest is Test {
         _storage.registerSupervisor(_proposalId, _SUPERVISOR, _OWNER);
         _storage.makeFinal(_proposalId, _SUPERVISOR);
         assertFalse(_storage.isExecuted(_proposalId));
-        _storage.setExecuted(_proposalId, _OWNER);
+        _storage.setExecuted(_proposalId);
         assertTrue(_storage.isExecuted(_proposalId));
     }
 
     function testSetExecutedNotOwner() public {
         _storage.registerSupervisor(_proposalId, _SUPERVISOR, _OWNER);
         _storage.makeFinal(_proposalId, _SUPERVISOR);
-        vm.expectRevert(abi.encodeWithSelector(Storage.NotSender.selector, _proposalId, _SUPERVISOR));
-        _storage.setExecuted(_proposalId, _SUPERVISOR);
+        vm.expectRevert("Ownable: caller is not the owner");
```

Comment:
confused here - if we're testing executed for not owners and we just removed req for the prop supervisor, why are we expecting a revert here?

might not understand fully how these tests work